internal/view/context.go

```diff
@@ -40,7 +41,10 @@ func NewContext(gvr *client.GVR) ResourceViewer {
 
 func (c *Context) bindKeys(aa *ui.KeyActions) {
 	aa.Delete(ui.KeyShiftA, tcell.KeyCtrlSpace, ui.KeySpace)
-	aa.Add(ui.KeyR, ui.NewKeyAction("Rename", c.renameCmd, true))
+	if !c.App().Config.IsReadOnly() {
```

Comment:
nit: we typically use these by defining bindDangerousKeys(). See examples from other views.

Reply:
Resolved in next commit :) Thanks @derailed